test/tools/check_compilation.py

```diff
@@ -101,7 +101,7 @@ def getLogErrors(log):
 			elif terminatedError.search(line):
 				terminated = True
 			else:
-				print line,
+				print (line, end=" ")
```

Comment:
This line shows issues with python 2.
  File "flang/test/f90_correct/lit/../../tools/check_compilation.py", line 104
    print (line, end=" ")

Reply:
`sys.stdout.write` would work.

Reply:
thanks for pointing it out. will fix it and submit new patch.